views/js/qtiCreator/widgets/interactions/textEntryInteraction/states/Question.js

```diff
@@ -29,12 +29,13 @@ define([
     var TextEntryInteractionStateQuestion = stateFactory.extend(Question);
 
     TextEntryInteractionStateQuestion.prototype.initForm = function(){
-        
+
         var _widget = this.widget,
             $form = _widget.$form,
             $inputs,
             interaction = _widget.element,
             patternMask = interaction.attr('patternMask'),
+            patternMaskMessage = interaction.attr('data-pattermask-message'),
```

Comment:
Why `data-pattermask-message` and not `patternMaskMessage` as prev `patternMask` ?

Reply:
+1

Reply:
@habemuscode @bziondik Good question, in the code I wrote it as `patternMaskMessage` because it is just variable, but according to discussions in the tickets, the attribute should be `data-pattermask-message` to comply with QTI standards. But I am not sure, I hope @julien-sebire could help.

Reply:
You are right, in discussions in ticket, we have `data-pattern-message`

So my doubt now is, the other attributes follow QTI standard? @julien-sebire